src/Nanoid/Nanoid.cs

```diff
@@ -1,107 +1,278 @@
 ﻿using System;
 using System.Runtime.CompilerServices;
-using System.Text;
 using System.Threading.Tasks;
 
 [assembly:InternalsVisibleTo("Nanoid.Test")]
 
 namespace NanoidDotNet
 {
     /// <summary>
-    /// 
+    /// Static class containing all functions and constants related to Nanoid.
     /// </summary>
     public static class Nanoid
     {
-        private const string DefaultAlphabet = "_-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
-        private static readonly CryptoRandom Random = new CryptoRandom();
         /// <summary>
-        /// 
+        /// Useful alphabets for Nanoid generation.
+        /// Taken from https://github.com/CyberAP/nanoid-dictionary
+        /// and https://github.com/SasLuca/zig-nanoid/blob/91e0a9a8890984f3dcdd98c99002a05a83d0ee89/src/nanoid.zig#L4.
         /// </summary>
-        /// <param name="alphabet"></param>
-        /// <param name="size"></param>
-        /// <returns></returns>
-        public static async Task<string> GenerateAsync(string alphabet = DefaultAlphabet, int size = 21)
+        public static class Alphabets

```

Comment:
A collection of useful nanoid alphabets, mostly taken from nanoid-dictionary. I don't think making a separate package for these would make sense. At best we could extra them into another file but I think they are fine sitting here.